DAO.sol

```diff
@@ -552,24 +552,24 @@ contract DAO is DAOInterface, Token, TokenCreation {
                 proposalCheck = false;
         }
 
-        // Execute result
-        if (quorum >= minQuorum(p.amount) && p.yea > p.nay && proposalCheck) {
+        if (quorum >= minQuorum(p.amount)) {
             if (!p.creator.send(p.proposalDeposit))
                 throw;
 
             lastTimeMinQuorumMet = now;
+            if (quorum > totalSupply / 5)
```

Comment:
Add a comment here stating that every time quorum reaches a 20% the minQuorumDivisor will be reset back to `5` making it back to 20% from whatever value it was before.